tasks.c

```diff
@@ -8492,7 +8519,13 @@ TickType_t uxTaskResetEventItemValue( void )
 
         traceENTER_ulTaskGetIdleRunTimePercent();
 
-        ulTotalTime = portGET_RUN_TIME_COUNTER_VALUE() * configNUMBER_OF_CORES;
+        #ifdef portALT_GET_RUN_TIME_COUNTER_VALUE
+            portALT_GET_RUN_TIME_COUNTER_VALUE( ulTotalTime );
+        #else
+            ulTotalTime = ( configRUN_TIME_COUNTER_TYPE ) portGET_RUN_TIME_COUNTER_VALUE();
+        #endif
+
+        ulTotalTime = ulTotalTime * configNUMBER_OF_CORES;
```

Comment:
This suggestion is added to align with the same syntax used on line [8531](https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/6e6e9347bc030997fa3d04cd4f85a958405a263a/tasks.c#L8531).
```suggestion
        ulTotalTime *=  configNUMBER_OF_CORES;
```

Reply:
Done.